about.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ABOUT | Flavors of Peru 2022</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="about-body">
+    <div id="navbar-menu" class="backdrop display-none">
+        
+        <a  id="exit-link-about" class="exit-link" onclick="close()">X</a>  
+        
+        <ul class="menu-items">
+            <li class="menu-list-option"><a href="index.html" class="menu-option">Home</a></li> 
+            <li class="menu-list-option"><a href="about.html" class="menu-option">About</a></li> 
+            <li class="menu-list-option"><a href="index.html#program" class="menu-option">Program</a></li> 
+            <li class="menu-list-option"><a href="index.html#speakers" class="menu-option">Speakers</a></li> 
+            <li class="menu-list-option"><a href="index.html#submit-button" class="menu-option">Join</a></li> 
+        </ul>
+
+    </div>
+    <header id="header" class="about-header header">
+           
+        <div id="navbar-container">
+
+            <nav id="desktop-navbar">
```

Comment:
- [ ] The navbar menu doesn't have the option to leave from this when I don't want to make any option. Please add a close option to the menu. 🌴 
![image](https://user-images.githubusercontent.com/20563252/137224002-ceeb374e-e5d6-4288-85ea-33cbf98ead11.png)